Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Devices/GoogleVR.cs

```diff
@@ -1,4 +1,4 @@
-#if ENABLE_VR && !DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR && !PACKAGE_DOCS_GENERATION
+#if UNITY_INPUT_SYSTEM_ENABLE_XR && !DISABLE_BUILTIN_INPUT_SYSTEM_GOOGLEVR && !PACKAGE_DOCS_GENERATION
```

Comment:
Not directly related to this PR but `&& !PACKAGE_DOCS_GENERATION` just caught my eye. That doesn't look right. Seems like that should be `(...) || PACKAGE_DOCS_GENERATION`.